feat(evs): support datasource to query evs quota details V3 (#7046)
huaweicloud/terraform-provider-huaweicloud: docs/data-sources/evsv3_quotas.md

---
subcategory: "Elastic Volume Service (EVS)"
layout: "huaweicloud"
page_title: "HuaweiCloud: huaweicloud_evsv3_quotas"
description: |-

>   Use this data source to get EVS quota details within HuaweiCloud.

Reviewer: ```suggestion
  Use this data source to get EVS v3 quota details within HuaweiCloud.
```

---

# huaweicloud_evsv3_quotas

Use this data source to get EVS quota details within HuaweiCloud.

## Example Usage

```hcl
variable "usage" {}

data "huaweicloud_evsv3_quotas" "test" {

>   usage             = var.usage

Reviewer: ```suggestion
  usage = var.usage
```

}
```

## Argument Reference

The following arguments are supported:

* `region` - (Optional, String) Specifies the region in which to query the resource.
  If omitted, the provider - level region will be used.

* `usage` - (Required, Boolean) Specifies whether to query quota details. Only value **true** is supported currently.

## Attribute Reference

In addition to all arguments above, the following attributes are exported:

* `id` - The data source ID of EVS quota details.

* `quota_set` - The returned quota information.
  The [quota_set](#quota_set_struct) structure is documented below.

<a name="quota_set_struct"></a>
The `quota_set` block supports:

* `backup_gigabytes` - The backup size, in GiB.

>   The [quota_params](#quota_params_struct) structure is documented below.

Reviewer: ```suggestion
  The [quota_set_sub](#quota_set_sub_struct) structure is documented below.
```
